### ISSUE TITLE
Foxy Gestures not working

### ISSUE THREAD

**Issue by CheshireSwift** [zen-browser/desktop#289]
### What happened?

The Foxy Gestures extension doesn't appear to work (the lines to show gestures are drawn, but the effects of the gestures don't occur). I'm aware some other Firefox-based browsers have struggled with gestures at times, but I'm not familiar with specifics.

I know not everyone makes use of gestures, but this is a make-or-break feature/compatibility for me.

### Reproducible?

- [x] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.0-a.18 (64-bit)

### Severity impact

Medium

### What platform are you seeing the problem on?

Windows

**Comment by enoversum** [comment]
I don't know if it's the same issue, but I use a system-wide gesture tool called StrokesPlus, and executing a gesture on top of Zen is bringing up a right-click menu instead of the gesture being executed. I would love to use gestures on Zen!

**Comment by enoversum** [comment]
@mauro-balades I don't want to rush you in any way, and we all know how busy you must be updating and improving this browser all the time. 
I just have no idea how trivial or complex this problem actually is. I'm a creature of habits, I'm so used to using gestures when browsing, because it's faster, this is literally the only reason I'm holding back on using Zen as my main browser. Can you give us a little update on this issue, even if it's not a (quick) fix?

**Comment by mauro-balades** [comment]
I can't right now, this afternoon I'll look into it

**Comment by enoversum** [comment]
@mauro-balades Did you find anything out? Or did it drown in all the other tasks and reports you're having?

**Comment by enoversum** [comment]
> @mauro-balades Did you find anything out? Or did it drown in all the other tasks and reports you're having?

Really not trying to push it, but this is literally the only reason why I can't switch to Zen as my default browser. If this is something more serious, I'm happy to wait though.

**Comment by enoversum** [comment]
> I can't right now, this afternoon I'll look into it

Any updates on this? It's literally the only thing making Zen my default browser.

**Comment by enoversum** [comment]
> Hi, @CheshireSwift. I'm helping the desktop team manage their backlog and am marking this issue as stale.
> 
> You've reported that the Foxy Gestures extension is not functioning as intended in Zen, with gesture lines visible but no effects occurring, while it works fine in Mozilla Firefox. Another user, enoversum, has also noted a similar issue with a system-wide gesture tool. Although maintainer mauro-balades acknowledged the problem and promised to investigate, there haven't been any updates yet.
> 
> Could you please let us know if this issue is still relevant to the latest version of the desktop repository? If it is, feel free to comment here to keep the discussion active. Otherwise, you can close the issue yourself, or it will be automatically closed in 7 days. Thank you!

Since I repeatedly tried to push this issue quite a few times - yes, this issue is very much still alive and raving. And this is probably the only issue hindering me on fully switching to Zen. So pretty please, dear bot, keep this issue alive. I'd so much love this to be solved.

**Comment by enoversum** [comment]
@mr-cheff The official Firefox does allow for gestures with a right- or middle-click by default, only Zen blocks it. All other browsers also allow it. As this ticket has been closed, will this not be fixed for Zen at all?

**Comment by enoversum** [comment]
@mauro-balades I would so much love to use Zen productively, yet I have no idea if this Zen-specific right-click bug is even likely to be solved. Or if it is even a bug, or something that could easily be switched on/off. Could you maybe give me a quick hint? Thank you so much in advance!

**Comment by mr-cheffy** [comment]
Does this still happen though?

**Comment by enoversum** [comment]
@mauro-balades Yes, of course, unfortunately. Try to hold and drag the middle or right mouse button, and you'll get a menu (right click) or see nothing. In both cases, the click and drag isn't recognised on Zen, so gestures fail. All other browsers, including Firefox, work fine.

**Comment by enoversum** [comment]
@mauro-balades Were you able to confirm this issue? Or do you need any more input?

**Comment by mr-cheffy** [comment]
I got it, I'm just focusing on other bugs

**Comment by enoversum** [comment]
@mauro-balades I was afraid you would say that. I'll send out a congratulation card on this bug's first birthday then :D ! 
No, in all seriousness, I understand you're trying to fix other bugs, is there any chance though you could take a closer look at this at some point? I'm still clinging to Brave, even though I love Zen a lot.

**Comment by enoversum** [comment]
@dosubot The issue persists and didn't solve itself, right-click and middle-click gestures do not work exclusively with Zen.

**Comment by mr-cheffy** [comment]
## Closing this issue as part of a repo cleanup

Due to technical-debt we had in the past, we are currently clearing out old, duplicate, stale, and non-actionable issues to improve issue tracking and make it easier to focus on fixing real bugs.

If this issue is still relevant, please open a new issue using our updated issue template, which will help us categorize and address it more efficiently.

Thanks for your understanding and for helping us improve the project! 🚀

This issue can be reopened at: https://github.com/zen-browser/desktop/issues/new?template=bug_report.yml&title=Foxy%20Gestures%20not%20working%0A
